_includes/footer.html

```diff
@@ -384,7 +384,7 @@ <h5 class="sub-header has-text-white">
                     {%- assign url_input = social.twitter -%}
                     {%- include functions/external_url.html -%}
                     <a {{anchor}} class="social-link padding--right" title="Twitter">
-                        <span class="sgds-icon sgds-icon-twitter is-size-4"></span>
+                        <img class="social-logo" src="/assets/img/x_logo.svg" alt="Open Government Products Logo">
```

Comment:
issue: Sadly OGP doesn't own twitter

Reply:
LOL

Reply:
oops thanks for catching https://github.com/isomerpages/isomerpages-template/pull/362/commits/1168a34d5dff096e0028dfbb2a651718952bc249

Reply:
nice catch lol